code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -497,7 +497,8 @@
 	upgrade_reagents = list(
 		/datum/reagent/consumable/banana,
 		/datum/reagent/consumable/berryjuice,
-		/datum/reagent/consumable/strawberryjuice
+		/datum/reagent/consumable/strawberryjuice,
+		/datum/reagent/consumable/vanilla
```

Comment:
No, these sort of contributions have been constantly decreasing the amount of reliance bartenders and cooks have toward botanists for the more complex drinks.
Like, no surprise botanists are too good **also** because there is no real concept of duty for most service jobs.